src/components/button-toggle/button-toggle.style.ts

```diff
@@ -40,7 +41,18 @@ const StyledButtonToggleContentWrapper = styled.div`
   height: 100%;
 `;
 
-const StyledButtonToggleLabel = styled.label`
+export interface StyledButtonToggleLabelProps {
+  /** buttonIcon to render. */
+  buttonIcon?: IconType;
+  /** Sets the size of the buttonIcon (eg. large) */
+  buttonIconSize?: ButtonToggleIconSizes;
```

Comment:
```suggestion
  buttonIconSize?: `small` | `large`;
```
Since `button-toggle-types.d.ts` is only one line, in my opinion I think we could remove the file and include the type values here unless you think otherwise